pkg/kclient/volumes_test.go

```diff
@@ -449,37 +404,36 @@ func TestAddPVCAndVolumeMount(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 
-			objectMeta := CreateObjectMeta(tt.podName, tt.namespace, tt.labels, nil)
-
-			podTemplateSpec := GeneratePodTemplateSpec(objectMeta, tt.containers)
-
-			err := AddPVCAndVolumeMount(podTemplateSpec, tt.volumeNameToPVCName, tt.componentAliasToVolumes)
+			containers, pvcVols, err := GetPVCVolAndVolMount(tt.containers, tt.volumeNameToPVCName, tt.componentAliasToVolumes)
 			if !tt.wantErr && err != nil {
-				t.Errorf("TestAddPVCAndVolumeMount.AddPVCAndVolumeMount() unexpected error %v, wantErr %v", err, tt.wantErr)
+				t.Errorf("TestGetPVCVolAndVolMount.AddPVCAndVolumeMount() unexpected error %v, wantErr %v", err, tt.wantErr)
```

Comment:
We should also rename the test function name to `TestGetPVCAndVolumeMount`

Reply:
updated!